config/initializers/permissions.rb

```diff
@@ -63,7 +63,7 @@
 
       map.permission :manage_user,
                      {
-                       users: %i[index show edit update],
+                       users: %i[index show edit update change_status change_status_info],
```

Comment:
Another instance where this controller action was not covered before and was secretly granted to all admins.